src/main/features/ccj/routes/check-and-send.ts

```diff
@@ -93,7 +94,8 @@ export default express.Router()
           await new DraftService().save(draft, user.bearerToken)
         }
 
-        await CCJClient.request(claim.externalId, draft, user)
+        const countyCourtJudgment = CCJModelConverter.convertForRequest(draft.document)
```

Comment:
Make sure `ccjType` is set properly for admissions/determination journey and not hardcoded as `DEFAULT`. Based on the journey, possible types are:
```
public enum CountyCourtJudgmentType {
    DEFAULT,
    ADMISSIONS,
    DETERMINATION
}
```

Reply:
https://github.com/hmcts/cmc-citizen-frontend/pull/1045#discussion_r227719652